### PR TITLE
helm mode: Add EKS test coverage

### PR DESCRIPTION
**Pull request by michi-covalent**
Ref: #1625

### CODE REVIEW

**Comment by michi-covalent** [comment]
✅ success: https://github.com/cilium/cilium-cli/actions/runs/4997969753

removing the temporary commit and marking it ready for review

File: install/install.go [hunk 1]
```diff
@@ -626,6 +626,18 @@ func (k *K8sInstaller) listVersions() error {
 	return err
 }
 
+func getChaningMode(values map[string]interface{}) string {
```
Comment (viktor-kurchenko) [inline install/install.go:629]: Typo: `getChaningMode ` -> `getChainingMode` ?

**Review by viktor-kurchenko — APPROVED** [review]
LGTM.

File: install/install.go (hunk 1 above)
Comment (michi-covalent) [reply install/install.go]: updated 👍

**Review by asauber — APPROVED** [review]

**Pull request merged**